src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -2323,13 +2338,16 @@ private IEnumerable<PSModuleInfo> CreateFakeModuleObject(IEnumerable<ModuleSpeci
             }
 
             // Process "FileList"
-            List<string> fileList;
-            if (
-                !GetListOfFilesFromData(data, moduleManifestPath, "FileList", manifestProcessingFlags, moduleBase, ""
-                    /*extension*/,
-                    false
-                    /* don't check file existence - don't want to change current behavior without feature team discussion */,
-                    out fileList))
+            if (!GetListOfFilesFromData(
+                data,
+                moduleManifestPath,
+                key: "FileList",
+                manifestProcessingFlags,
+                moduleBase,
+                extension: "",
```

Comment:
I thought we replaced all "" with `string.Empty`. It is not request for the PR.

Reply:
Maybe we can use [RCS1078i](https://github.com/JosefPihrt/Roslynator/blob/master/docs/analyzers/RCS1078i.md) analyzer.

Reply:
I will submit PR for [SA1122](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1122.md).